streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -243,4 +247,27 @@ public void to(String topic, Serializer<K> keySerializer, Serializer<V> valSeria
         return new KTableImpl<>(topology, joinMergeName, joinThis, allSourceNodes);
     }
 
+    @Override
+    public <K1, V1, V2> KTable<K1, V2> aggregate(AggregatorSupplier<K1, V1, V2> aggregatorSupplier, KeyValueMapper<K, V, KeyValue<K1, V1>> selector, String name) {
+        // TODO
+        return null;
+    }
+
+    @Override
+    public <K1, V1> KTable<K1, Long> sum(KeyValueMapper<K, V, KeyValue<K1, V1>> selector, ValueMapper<V1, Long> valueMapper, String name) {
```

Comment:
Why do we need a valueMapper? can't we have just a selector of KeyValue<K1, Long>?